frontend/cypress/integration/featureFiles/kiali_logout.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Kiali logout
+
+  User wants to login to Kiali and see landing page
+  
+  @smoke
+  Scenario: Kiali logout successfully
+    Given user opens base url
+    And user clicks Log In With OpenShift
+    And user clicks my_htpasswd_provider
+    And user fill in username and password
+    And user see console in URL
```

Comment:
Sorry I did not noticed earlier: This is obsolete code, should be replaced with 
`Given user is at administrator perspective`

Reply:
@ScriptingShrimp : It is taken last commit, I have update only ts file. I will update this file as well. Thanks for the review.